application.py

```diff
@@ -4,26 +4,28 @@
 import os
 
 import newrelic.agent  # See https://bit.ly/2xBVKBH
-import sentry_sdk
+
+# import sentry_sdk
 from apig_wsgi import make_lambda_handler
 from dotenv import load_dotenv
 from flask import Flask
-from sentry_sdk.integrations.celery import CeleryIntegration
-from sentry_sdk.integrations.flask import FlaskIntegration
-from sentry_sdk.integrations.redis import RedisIntegration
-from sentry_sdk.integrations.sqlalchemy import SqlalchemyIntegration
+
+# from sentry_sdk.integrations.celery import CeleryIntegration
+# from sentry_sdk.integrations.flask import FlaskIntegration
+# from sentry_sdk.integrations.redis import RedisIntegration
+# from sentry_sdk.integrations.sqlalchemy import SqlalchemyIntegration
 from werkzeug.middleware.proxy_fix import ProxyFix
 
 from app import create_app
 
 load_dotenv()
 
-if "SENTRY_URL" in os.environ:
-    sentry_sdk.init(  # type: ignore
-        dsn=os.environ.get("SENTRY_URL", ""),
-        integrations=[CeleryIntegration(), FlaskIntegration(), RedisIntegration(), SqlalchemyIntegration()],
-        release="notify-api@" + os.environ.get("GIT_SHA", ""),
-    )
+# if "SENTRY_URL" in os.environ:
+#     sentry_sdk.init(  # type: ignore
+#         dsn=os.environ.get("SENTRY_URL", ""),
+#         integrations=[CeleryIntegration(), FlaskIntegration(), RedisIntegration(), SqlalchemyIntegration()],
+#         release="notify-api@" + os.environ.get("GIT_SHA", ""),
+#     )
```

Comment:
Sentry causes failures due to our old eventlet. We are not using sentry in production. Will discuss removing offline.

Reply:
After talking IRL got rid of sentry